_data/glossary/container-env-variables.yaml

```diff
@@ -0,0 +1,9 @@
+id: container-env-variables
+name: Container Environment Variables
+full-link: /docs/concepts/containers/container-environment-variables.md
+tags:
+- fundamental
+short-description: >
+  Container environment variables are name/value pairs that provide useful information to the container runtime.
+long-description: >
+  Container environment variables provide important resources to [Containers] {% glossary_tooltip text="Containers" term_id="container" %} such as file system, information about container itself and other cluster resources.
```

Comment:
> about container itself

about the container itself?

Reply:
I have tried to used 'container running in the Pod' in place of container runtime. Please let me know if that is acceptable.